checker/tests/resourceleak-permitinitializationleak/InstanceInitializer.java

```diff
@@ -0,0 +1,47 @@
+// A test that the checker is sound in the presence of instance initializer blocks.
```

Comment:
A comment in this test should call out exactly where and how the unsoundness occurs.